input/groups/armor.json

```diff
@@ -2573,6 +2566,26 @@
         10198,
         10211
       ],
+      "Heritage": [
```

Comment:
Yeah, it's an external product, so it shouldn't be at Core. I only realized it was an issue recently, talking about how Steam users will never be able to earn those in any way whatsoever, since they can't connect GW1 accounts.

I agree Gem Store has way too many things that aren't really gem store, but since most of them are redeemed through gem store codes... I wouldn't change the name though, maybe a separate "Promotional" category?

Things that could go into the new category:

* Hall of Monuments
* Purchase Bonus with no alternative acquisition (preorder bonus + funkos)
* Promotional Event with no alternative acquisition

I thought there wouldn't be enough meat, but the numbers look pretty well, even without the GW1 stuff:

* 12 armor pieces
* 3 back items
* 6 outfits
* 1 glider
* 1 novelty
* 8 miniatures

I'll think about it, I've recently done a new category for special events, I'll do another for these, then do another pull request with both next week or so, once this one goes through.